components/SocialShareButtons/index.tsx

```diff
@@ -0,0 +1,40 @@
+import Image from 'next/image';
+import {
+  FacebookShareButton,
+  LinkedinShareButton,
+  TwitterShareButton
+} from 'react-share';
+
+const SocialIcon = ({ src, alt }: { src: string; alt: string }) => (
+  <div className="rounded-lg border border-gray-300 p-3 shadow-sm shadow-blue-900/5 hover:border-gray-400">
+    <Image className="saturate-0" width={20} height={20} src={src} alt={alt} />
+  </div>
+);
+
+const SocialShareButtons = ({ title, url }: { title: string; url: string }) => (
```

Comment:
ShareButton should be generic. You should create responsive generic social button.